backend/src/data/repositories/course/course.repository.ts

```diff
@@ -77,7 +80,9 @@ class Course {
     const { results, total } = await this.#CourseModel
       .query()
       .withGraphJoined('category')
-      .orderBy('courseCategoryId', SortOrder.DESC)
+      .orderByRaw(
+        `${Course.CATEGORY_ID_COLUMN_NAME} ${SortOrder.ASC} NULLS FIRST`,
```

Comment:
You could snakeCase this col by existing helper without declaring a separate static var for it

Reply:
@andanfe do you mean by using `changeStringCase`? In that case, we'll still have to create a variable for `"courseCategoryId"`, to use inside `changeStringCase`

Reply:
You could call it just inside template string, but in that case you'd also could provide `course_category_id` inside orderByRaw query query 🤔 Just not sure if you need to define static var for this single query :\

Reply:
I also was thinking about using `course_category_id` inside of query, but changed my mind, because in other places we use camelCase, didn't want to have only one snake_case in this query